app/src/main/kotlin/com/wire/android/ui/authentication/login/sso/LoginSSOViewModel.kt

```diff
@@ -134,7 +136,9 @@ class LoginSSOViewModel @Inject constructor(
             val defaultAuthScope: AuthenticationScope =
                 coreLogic.versionedAuthenticationScope(
                     authServerConfigProvider.defaultServerLinks()
-                )().let {
+                    // domain lockup does not support proxy
+                    // TODO: add proxy support
```

Comment:
For later?

Reply:
domain lookup happen on wire servers and the result of it is a new on prem server config that the user can login with.
for the time being proxy is not supported with that feature but the result server config can have a vproxy and it works just fine